virttest/utils_v2v.py

```diff
@@ -1836,8 +1836,37 @@ def compare_version(interval, version=None, cmd=None):
     """
     if not version:
         if not cmd:
-            cmd = 'rpm -q libguestfs'
+            cmd = 'rpm -q virt-v2v'
```

Comment:
Is it required to distinguish the rpm based on the release version?

Reply:
No, This is a default behaviour change. 
Users should always tell the api what cmd they are using like the new function does.

Reply:
Ok, got it. Thank you.